Sources/Afterpay/Model/PriceBreakdown.swift

```diff
@@ -25,20 +25,22 @@ struct PriceBreakdown {
     showWithText: Bool = true
   ) {
     let configuration = getConfiguration()
+
+    let greaterThanZero = totalAmount > .zero
+    let greaterThanOrEqualToMinimum = totalAmount >= (configuration?.minimumAmount ?? .zero)
+    let lessThanOrEqualToMaximum = totalAmount <= (configuration?.maximumAmount ?? .zero)
+    let inRange = greaterThanZero && greaterThanOrEqualToMinimum && lessThanOrEqualToMaximum
+    let maxFractionDigits = !inRange ? 0 : nil
+
     let formatter = configuration
       .map { CurrencyFormatter(locale: $0.locale, currencyCode: $0.currencyCode, clientLocale: Locale.current) }
-    let format = { formatter?.string(from: $0) }
+    let format = { formatter?.string(from: $0, maxDecimals: maxFractionDigits) }
 
-    let formattedMinimum = configuration?.minimumAmount.flatMap(format) ?? formatter?.string(from: 1)
+    let formattedMinimum = configuration?.minimumAmount.flatMap(format) ?? formatter?.string(from: 1, maxDecimals: maxFractionDigits)
```

Comment:
Can this be split into multiple lines somehow - to keep the linter happy?

Reply:
Addressed in [d3a89ba](https://github.com/afterpay/sdk-ios/pull/226/commits/d3a89ba5235d5b5b1f18565e90ccc8aabb55c44e)